spec/migrations/20160119125950_add_created_on_for_container_entities_spec.rb

```diff
@@ -0,0 +1,82 @@
+require_migration
+
+def stub_name_for(table)
+  "#{table}_stub"
+end
+
+def stub_for(table)
+  send(stub_name_for(table))
+end
+
+describe AddCreatedOnForContainerEntities do
+  CONTAINER_TABLES = [:container_nodes, :container_projects, :container_services, :container_routes, :container_groups,
+                      :container_replicators, :container_quotas, :container_builds, :container_build_pods,
+                      :container_limits, :container_volumes].freeze
+
+  CONTAINER_TABLES.each { |table| let(stub_name_for(table)) { migration_stub(table.to_s.classify.to_sym) } }
+
+  let(:mock_timestamp) { Time.zone.parse('2016-03-09 12:00:38.711120') }
+  let(:mock_name)      { "Name_1" }
+
+  migration_context :up do
+    it "creates column created_on and created_on and ems_created_on have same values as creation_timestamp" do
+      records = {}
+
+      CONTAINER_TABLES.each do |table|
+        records[table] = stub_for(table).create!(:name => mock_name, :creation_timestamp => mock_timestamp)
+      end
+
+      migrate
+
+      CONTAINER_TABLES.each do |table|
+        expect(records[table].reload).to have_attributes(:created_on => mock_timestamp)
+        expect(records[table].reload).to have_attributes(:ems_created_on => mock_timestamp)
+      end
+    end
+
+    it "removes column creation_timestamp" do
+      records = {}
+
+      CONTAINER_TABLES.each do |table|
+        records[table] = stub_for(table).create!(:name => mock_name)
+      end
+
+      migrate
+
+      CONTAINER_TABLES.each do |table|
+        expect(records[table].reload).not_to have_attributes(:creation_timestamp => nil)
+      end
+    end
+  end
+
+  migration_context :down do
+    it "renames ems_created_on back to creation_timestamp and it has same value as ems_created_on" do
+      records = {}
+
+      CONTAINER_TABLES.each do |table|
+        records[table] = stub_for(table).create!(:name => mock_name, :ems_created_on => mock_timestamp)
+      end
+
+      migrate
+
+      CONTAINER_TABLES.each do |table|
+        expect(records[table].reload).to have_attributes(:creation_timestamp => mock_timestamp)
+        expect(records[table].reload).not_to have_attributes(:ems_created_on => mock_timestamp)
```

Comment:
yes ok, renaming column and removing is rails-embeded, so I agree that we do not need to test it :+1: